docs/modules/ROOT/pages/erc1155-pausable.adoc

```diff
@@ -0,0 +1,129 @@
+= ERC-1155 Pausable
+
+ERC1155 token with pausable token transfers, minting, and burning.
+
+Useful for scenarios such as preventing trades until the end of an evaluation period, or having an emergency switch for freezing all token transfers in the event of a large bug.
+
+[[usage]]
+== Usage
+
+In order to make your ERC1155 token `pausable`, you need to use the https://docs.rs/openzeppelin-stylus/0.2.0-alpha.2/openzeppelin_stylus/utils/pausable/index.html[`Pausable`] contract and apply its mechanisms to ERC1155 token functions as follows:
+
+[source,rust]
+----
+use openzeppelin_stylus::{
+    token::erc1155::{Erc1155, IErc1155},
+    utils::Pausable,
+};
+
+sol_storage! {
+    #[entrypoint]
+    struct Erc1155Example {
+        #[borrow]
+        Erc1155 erc1155;
+        #[borrow]
+        Pausable pausable;
+    }
+}
+
+#[public]
+#[inherit(Erc1155, Pausable)]
+impl Erc1155Example {
+    pub fn mint(
```

Comment:
@qalisander @bidzyyys this example relates closely to [ERC721 Pausable refactor PR](https://github.com/OpenZeppelin/rust-contracts-stylus/pull/414#issuecomment-2498855455) - if we decide to take the route that you used here, then we should close the PR I linked

Reply:
@0xNeshi please moderate this discussion on Slack

Reply:
Right now it is consistent with Erc721Burnable and Erc20Burnable